apps/emqx_plugin_libs/src/emqx_plugin_libs_ssl.erl

```diff
@@ -96,8 +96,19 @@ do_save_file(FilePath, _) when is_list(FilePath) ->
     FilePath;
 do_save_file(_, _) -> "".
 
-do_save_file("", _, _Dir) -> ""; %% ignore
-do_save_file(_, <<>>, _Dir) -> ""; %% ignore
+do_save_file("", _, _Dir) -> ""; %% no filename, ignore
+do_save_file(FileName, <<>>, Dir) ->  %% no content, see if file exists
+    %% when FileName is an absolute path, filename:join has no effect
+    %% hence a usort.
+    {ok, Cwd} = file:get_cwd(),
+    Possibles = lists:usort(
+                  [filename:join([Cwd, FileName]), %% relative to cwd
```

Comment:
if L106 is for management API upload, what is this L105 for?

Reply:
105 is for user filling in a pre-exiting file (can be relative path) from UI without uploading a file